xwiki-pro-macros-api/src/main/resources/templates/userlist/userlist.vm

```diff
@@ -0,0 +1,17 @@
+#set ($users = $params.users)
+#set ($users = $users.split(','))
+#set ($discard = $xwiki.ssx.use('XWiki.Macros.UserListMacroStyleSheet'))
```

Comment:
This makes the API module depend on the UI (which depends on the API). Would be better to have the CSS either in the API module (``ssrx``) or in a new WebJar module.

Reply:
Now using a CSS via `ssrx` indeed.